po/it.po

```diff
@@ -484,15 +491,15 @@ msgstr "_Elimina"
 
 #: data/menu.ui.h:19
 msgid "Dupl_icate"
-msgstr "D_uplica"
+msgstr "Dupl_ica"
 
 #: data/menu.ui.h:20
 msgid "Re_verse Order"
-msgstr "_Inverti Ordine"
+msgstr "In_verti ordine"
 
```

Comment:
Please explain this change, it's not necessarily wrong, I just want to be sure you know what you are doing here.

Reply:
"pick a strong letter (e.g. in "Search and replace" rather pick s, r or p than a)"

Reply:
> It looks like you are changing the grammar here...

Here must be "Invertire ordine"?